### ISSUE TITLE
Twig block sylius_resource_autocomplete_choice_row may implement "help" attribute

### ISSUE THREAD

**Issue by Geolim4** [Sylius/Sylius#15450]
**Describe the proposed solution**
To allow better form customization without having to override this block and the template `@SyliusUi/Form/theme.html.twig` it would be nice to be able to add support for the "help" attribute.

**Describe alternatives you've considered**
Currently needed to override the block like this:
```twig
{# @LocalBundle/Form/theme.html.twig #}
{% extends '@!SyliusUi/Form/theme.html.twig' %}

{% block sylius_resource_autocomplete_choice_row %}
    {{ parent() }}
    {% if help is defined %} <div class="any-good-css-class">{{ help }}</div> {% endif %}
{% endblock %}

```

**Additional context**
N/A

**Comment by crydotsnake** [comment]
Hello @Geolim4 !

Where in the backend is this block/view used? 🤔

**Comment by Geolim4** [comment]
Hello @crydotsnake almost everywhere there's an autocomplete field (taxons/product/variants fields) :D

**Comment by crydotsnake** [comment]
Found it ;)

And the `Help` attribute in general i assume would be a help message for the editor? Cant find anything about this in the documentation 🤔

**Comment by Geolim4** [comment]
Yeah it would work a small text under the label to add a brief description.

**Comment by crydotsnake** [comment]
You mean you would like to configure a help text for those two labels seperatley?

![SCR-20240227-oays](https://github.com/Sylius/Sylius/assets/39345336/b5acae25-274c-4145-85ee-201769364c9d)

In case you have a screenshot how it looks like in your project and would like to show it, that would be very helpful! That I can imagine it better :)

**Comment by crydotsnake** [comment]
Nvm.

Its only the "options" dropdown.

![SCR-20240227-oesu](https://github.com/Sylius/Sylius/assets/39345336/6d05d2d4-9694-4bfd-a7b3-ed22c8933296)

**Comment by Geolim4** [comment]
Its the autocomplete-based dropdown indeed which does not (yet) support implementation of the "help" attribute.
![image](https://github.com/Sylius/Sylius/assets/1332071/ca813dd7-92c2-451c-ac64-50d954eae97a)

My code snippet above allow the implementation using a small twig tweak.

**Comment by crydotsnake** [comment]
Ahh. `help` comes from the PHP controller and can be used in Twig as a variable then 🙂💡

**Comment by Geolim4** [comment]
Not exactly, it comes from the Symfony form builder base options 😁

**Comment by crydotsnake** [comment]
Okay. Understood. But wouldn't you then have the same label in every place? I understood that you want to have a different one in every place where this template is used.

**Comment by Geolim4** [comment]
Its because the "help" option is mostly well configured on all fields from the admin area, except the auto-complete based field based on `sylius_resource_autocomplete_choice_row` block :D
